pkg/controllers/vdb/onlineupgrade_reconciler.go

```diff
@@ -278,6 +297,11 @@ func (r *OnlineUpgradeReconciler) postCreateNewSubclustersMsg(ctx context.Contex
 // exists in the main cluster. This is a pre-step to setting up replica group B, which will
 // eventually exist in its own sandbox.
 func (r *OnlineUpgradeReconciler) assignSubclustersToReplicaGroupB(ctx context.Context) (ctrl.Result, error) {
+	// If we have already promoted sandbox to main, we don't need to do this step
+	if vmeta.GetOnlineUpgradeSandboxPromoted(r.VDB.Annotations) == vmeta.SandboxPromotedTrue {
```

Comment:
Move this  to vdb: `vdb.IsOnlineUpgradeSandboxPromoted()`.

Reply:
All get annotations methods are inside annotations.go. Why do we need to move them to vdb?